conf/variant/common/local.conf

```diff
@@ -17,3 +17,6 @@ BB_DANGLINGAPPENDS_WARNONLY = "1"
 
 # Fix for rpm metadata clash between nativesdk-cmake and nativesdk-qtbase-tools
 DIRFILES_pn-nativesdk-cmake = "1"
+
+# Don't install any extra devel packages just because we're running on QEMU
```

Comment:
I'm assuming "devel" is short for "development", but why not at least write out the word in the comment?
If there some (official) background reason to this decision could you please include it in the commit message (not on the first line).

Reply:
I just tried staing < 80 chars in width ;)